### PR TITLE
[CUBRIDMAN-59] Change the result type of ROWNUM and INST_NUM() from INT to BIGINT

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CUBRIDMAN-59

Revise the return type of ROWNUM, ORDERBY_NUM, GROUPBY_NUM from INT to BIGINT
- revise return type, description, and examples

### CODE REVIEW

File: en/sql/function/rownum_fn.rst [hunk 1]
```diff
@@ -15,11 +15,11 @@ ROWNUM, INST_NUM
...
 
-    :rtype: INT
+    :rtype: BIGINT
 
 **ROWNUM** and **INST_NUM()** can be used in the **SELECT** statement; **ORDERBY_NUM()** can be used in the **SELECT** statement with **ORDER BY** clauses, and **GROUPBY_NUM()** can be used in the **SELECT** statement with **GROUP BY** clauses. The **ROWNUM** function can be used to limit the number of result records of the query in several ways. For example, it can be used to search only the first 10 records or to return even or odd number records.
 
-The **ROWNUM** function has a result value as an integer, and can be used wherever an expression is valid such as the **SELECT** or **WHERE** clause. However, it is not allowed to compare the result of the **ROWNUM** function with the attribute or the correlated subquery.
+The **ROWNUM** function has a result value as an big integer, and can be used wherever an expression is valid such as the **SELECT** or **WHERE** clause. However, it is not allowed to compare the result of the **ROWNUM** function with the attribute or the correlated subquery.
```
Comment (beyondykk9) [inline en/sql/function/rownum_fn.rst]:
```suggestion
The **ROWNUM** function has a result value as a big integer, and can be used wherever an expression is valid such as the **SELECT** or **WHERE** clause. However, it is not allowed to compare the result of the **ROWNUM** function with the attribute or the correlated subquery.
```

File: ko/sql/function/rownum_fn.rst [hunk 2]
```diff
@@ -15,11 +15,11 @@ ROWNUM, INST_NUM
 
     **ROWNUM** 함수는 질의 결과로 생성될 각 레코드에 대한 순서를 나타내는 번호를 반환한다. 첫 번째 결과 레코드는 1, 두 번째 결과 레코드는 2를 가진다.
 
-    :rtype: INT
+    :rtype: BIGINT
 
 일반적인 **SELECT** 문에서는 **ROWNUM**, **INST_NUM()**\을, **ORDER BY** 절을 포함한 **SELECT** 문에서는 **ORDERBY_NUM()**\을, **GROUP BY** 절을 포함한 **SELECT** 문에서는 **GROUPBY_NUM()**\을 사용할 수 있다. **ROWNUM** 함수를 사용하면 질의의 결과 레코드 수를 다양한 방법으로 제한할 수 있다. 예를 들어, 질의 결과의 처음 10건만 조회한다거나, 짝수 번째 또는 홀수 번째 레코드만 반환하도록 할 수 있다.
 
-**ROWNUM** 함수는 결과 타입이 정수형이고, **SELECT** 절과 **WHERE** 절과 같이 질의 내에 수식이 위치할 수 있는 모든 곳에 사용할 수 있다. 하지만, **ROWNUM** 함수 결과를 속성 또는 연관된 부질의(correlated subquery)와 비교하는 것은 허용되지 않는다.
+**ROWNUM** 함수는 결과 타입이 큰 정수형이고, **SELECT** 절과 **WHERE** 절과 같이 질의 내에 수식이 위치할 수 있는 모든 곳에 사용할 수 있다. 하지만, **ROWNUM** 함수 결과를 속성 또는 연관된 부질의(correlated subquery)와 비교하는 것은 허용되지 않는다.
```
Comment (beyondykk9) [inline ko/sql/function/rownum_fn.rst:22]:
```suggestion
**ROWNUM** 함수는 결과 타입이 64비트 정수형이고, **SELECT** 절과 **WHERE** 절과 같이 질의 내에 수식이 위치할 수 있는 모든 곳에 사용할 수 있다. 하지만, **ROWNUM** 함수 결과를 속성 또는 연관된 부질의(correlated subquery)와 비교하는 것은 허용되지 않는다.
```

Comment (hgryoo) [reply ko/sql/function/rownum_fn.rst:22]: In the data types section, BIGINT is already translated as "큰 정수".
https://www.cubrid.org/manual/ko/11.0/sql/datatype.html#bigint

File: en/sql/function/rownum_fn.rst (hunk 1 above)
Comment (hgryoo) [reply en/sql/function/rownum_fn.rst]: Thank you

**Review by beyondykk9 — APPROVED** [review]

**Review by shparkcubrid — APPROVED** [review]

**Review by youngjinj — APPROVED** [review]

**Review by ctshim — APPROVED** [review]

**Review by hyunikn — APPROVED** [review]

**Pull request merged**